src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt

```diff
@@ -121,8 +121,8 @@ class HomeSettingsFragment : Fragment() {
                 }
             ),
             HomeSetting(
-                R.string.system_files,
-                R.string.system_files_description,
+                R.string.setup_system_files,
+                R.string.setup_system_files_description,
                 R.drawable.ic_system_update,
                 {
```

Comment:
We should probably move this setting underneath "Install CIA File". I find it unlikely that people will be setting up their system files as often as they'll be installing applications.